Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -2041,6 +2053,21 @@ private extension Purchases {
         }
     }
 
+    #if DEBUG
```

Comment:
Same here. I'd add the `!ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION` check just to be sure